runtime/src/lib.rs

```diff
@@ -118,7 +123,7 @@ impl sdk::Runtime for Runtime {
                 parameters: modules::core::Parameters {
                     min_gas_price: { BTreeMap::from([(Denomination::NATIVE, 100_000_000_000)]) },
                     max_batch_gas: if is_testnet() { 30_000_000 } else { 10_000_000 },
-                    max_tx_size: 32 * 1024,
+                    max_tx_size: u32::max_value(),
```

Comment:
This should be reduced to something more reasonable?

Reply:
AFAIK there is a limit for p2p propagation? If you rely on gas alone then the per-byte cost needs to be increased. I would just set this to something like 300 KB or so?

Reply:
I haven't encountered the p2p propagation issue in the same way that I haven't run out of gas due to tx size alone. 300 KiB is probably fine. I'd prefer to have cheap txs in the common case anyway.